deploy/csi/rbd/pvc.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: rbd-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  volumeMode: Block
```

Comment:
I have `losetup -j` issue when mounting `volumeMode: Block pvc`
I deployed minikube using 'virtualbox'

Reply:
In most cases, `rbd` is used for `block`, so it seems that `volumeMode` should be `Block`.
`losetup -j` issue seems to be caused by the virtualbox environment.